dace/transformation/interstate/sdfg_nesting.py

```diff
@@ -628,7 +629,7 @@ def _modify_access_to_access(self,
                                                                   matching_edge.data,
                                                                   use_dst_subset=True)
                             new_memlet = in_memlet
-                            new_memlet.other_subset = out_memlet.dst_subset
+                            new_memlet.other_subset = out_memlet.subset
```

Comment:
Is that not less safe? Which case does this pertain to?

Reply:
The problem arose in a (new) test, where it turned out that `dst_subset` was None. For outgoing edges (line 658 on the same file), we use `subset` (not `src_subset`), and I believe the original code above is just wrong.